webpack/assets/javascripts/foreman_hosts.js

```diff
@@ -0,0 +1,39 @@
+export let pluginEditAttributes = {
```

Comment:
this object is not imported anywhere, it is not used.

this can be replace with a comment on how to use window.tfm.hosts.pluginEditAttributes by supplying arrays.